src/jit/CMakeLists.txt

```diff
@@ -6,11 +6,12 @@ include_directories("../inc")
 # Enable the following for UNIX altjit on Windows
 # add_definitions(-DALT_JIT)
 
-if (WIN32) 
 if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
-  add_definitions(-DFEATURE_SIMD -DFEATURE_AVX_SUPPORT) 
-endif (CLR_CMAKE_PLATFORM_ARCH_AMD64)
+  add_definitions(-DFEATURE_SIMD) 
```

Comment:
I have manually checked the code generated for some of the SIMD unit tests, but that is all. This issue is related but orthogonal to this PR.